docs/EN_US/DataPatterns/DataPatternsMods/DataPatternsInECLWatch.xml

```diff
@@ -1,18 +1,178 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!DOCTYPE sect1 PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN"
+<!DOCTYPE chapter PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN"
 "http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd">
 <chapter id="DPinECLWatch">
   <title><emphasis>Data Patterns In ECL Watch </emphasis></title>
 
-  <para>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
-  eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim
-  veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea
-  commodo consequat.</para>
+  <para>In this example, we'll use a file created by GenData.ecl from the
+  Programmer's Guide. </para>
 
-  <para>Duis aute irure dolor in reprehenderit in voluptate velit esse cillum
-  dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
-  proident, sunt in culpa qui officia deserunt mollit anim id est
-  laborum.</para>
+  <orderedlist>
+    <listitem>
+      <para>Check to see that the container is executing without
+      issues.<programlisting>kubectl get pod</programlisting></para>
+    </listitem>
 
-  <para></para>
+    <listitem>
+      <para>Open ECL Watch in your browser using the following URL:
+      http://nnn.nnn.nnn.nnn:pppp (where nnn.nnn.nnn.nnn is your ESP Server's
+      IP Address and pppp is the port. The default port is 8010).</para>
+    </listitem>
+
+    <listitem>
+      <para>To access a file, click the third icon from the top, located on
+      the menu bar on the left-most side of the ECL Watch page.</para>
+
+      <graphic fileref="../../images/DP007.jpg"/>
+    </listitem>
+
+    <listitem>
+      <para>Double click the data file to access it.</para>
```

Comment:
Find the file you want, then double-click it to open the Logical Files Details page.